decidim-ai/spec/lib/engine_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Ai
+    describe ".resource_classifier" do
+      subject { Decidim::Ai::SpamDetection.resource_classifier }
+
+      it "returns a spam detection service" do
+        expect(subject).to be_a(Decidim::Ai::SpamDetection::Service)
+      end
+    end
+
+    describe ".user_classifier" do
+      subject { Decidim::Ai::SpamDetection.user_classifier }
+
+      it "returns a spam detection service" do
+        expect(subject).to be_a(Decidim::Ai::SpamDetection::Service)
+      end
+    end
+
+    describe ".language_detection_service" do
```

Comment:
It's probably a dumb question, but why are we adding this method? Is it actually being used somewhere?

Reply:
Is not a dumb question. In the original implementation  we had a `Decidim::AI.language_detection_service` which has been moved to :  `Decidim::AI::Language.service`

Reply:
will remove in the next commit.

Reply:
Let me rephrase the question: is the feature for language detection actually being used?

Reply:
Marking as solved as it was already handled